core/domain/stats_jobs_one_off_test.py

```diff
@@ -45,13 +45,17 @@ def run_one_off_job(self):
 
         Assumes the existence of a class constant ONE_OFF_JOB_CLASS, pointing
         to the queue under test.
+
+        Returns:
+            *. The output of the one off job.
```

Comment:
The type is known, right?

Reply:
Not for all jobs, I guess?

Reply:
Hm, the docstring in JobModel is ambiguous, though I always thought it was a string (or list of strings). OK, let's leave this as-is then.